src/utils/localization-utils.tsx/vacation-request-type-utils.tsx

```diff
@@ -0,0 +1,28 @@
+import { VacationType } from "generated/client";
+import strings from "localization/strings";
+
+/**
+ * Handle request type
```

Comment:
Could this be more descriptive

Reply:
Wrote a new description:
```suggestion
/**
 * Get request type as an enum case and return a corresponding localized string
 *
 * @param type Type of vacation, an enum case
 */
const getLocalizedRequestType = (type: VacationType) => { ...
```